articles/service-fabric/service-fabric-cluster-resource-manager-cluster-description.md

```diff
@@ -358,7 +358,7 @@ To support these sorts of configurations, Service Fabric includes tags that you
 </center>
 
 ### Built-in node properties
-Service Fabric defines some default node properties that can be used automatically so you don't have to define them. The default properties defined at each node are **NodeType** and **NodeName**. 
+Service Fabric defines some default node properties that can be used automatically so you don't have to define them. The default properties defined at each node are **NodeType**, **NodeName**, and **NodeTypeName**. 

```

Comment:
NodeTypeName is unfortunately not actually provided by the SF runtime. This is added on in Azure (and I believe SF Standalone, though I haven't looked recently) for historical reasons. Ex: If you stand up your local dev cluster and make no other modifications to the setup, only NodeType and NodeName will work, as those as the properties explicitly understood by the SF runtime. NodeTypeName is effectively a custom property. Also I don't know but we keep talking about converting it into a built in property and then not setting it as custom anymore, in order to make this less confusing. 

Given all this, what do you think we want to do here? Maybe add some separate section that today an additional property called NodeTypeName is available in Azure but that people are recommended to use NodeType since it's the real one?